tests/msc2716_test.go

```diff
@@ -112,43 +112,53 @@ func TestBackfillingHistory(t *testing.T) {
 			eventIDsAfter := createMessagesInRoom(t, alice, roomID, 2)
 
 			// Insert the most recent chunk of backfilled history
-			insertTime1 := timeAfterEventBefore.Add(timeBetweenMessages * 3)
+			insertTime0 := timeAfterEventBefore.Add(timeBetweenMessages * 3)
```

Comment:
Updating from `insertTime1` -> `insertTime0` to match the chunk explanation below